### PR TITLE
Add some better documentation and bug fixes

### PR DESCRIPTION
**Pull request by VSEphpbb**
This adds title tags to the "components" in the front end, with deeper explanations of what they will generate when selected.

Depends on https://github.com/phpbb/documentation/pull/80
Actually this doesn't really depend on the PR above, but it expects it to be merged some day. Shouldn't hold up this PR at all though.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,45 +2,16 @@
 
 phpBB's Skeleton Extension is a tool for extension authors to help speed up and simplify the task of starting a new extension project. It generates sample starter files and directories in a skeleton package that you can use to begin building your extension.
 
-## Requirements
+View the [documentation](https://area51.phpbb.com/docs/dev/31x/extensions/phpbb_skeleton_extension.html) for installation and usage information.
```
Comment (3D-I) [inline README.md]: Actually ATM this link leads to the main page at area51.com.

Comment (VSEphpbb) [inline README.md]: Depends on phpbb/documentation#80

Comment (3D-I) [inline README.md]: Fine, thx.

**Comment by VSEphpbb** [comment]
@michaelcullum

**Pull request merged**